pkg/apis/bundle/v1alpha1/object_template_types.go

```diff
@@ -23,26 +23,44 @@ import (
 type TemplateType string
 
 const (
-	// UndefinedTemplateType represents an undefined template type, and should
-	// default to GoTemplate.
+	// UndefinedTemplateType represents an undefined template type.
 	UndefinedTemplateType TemplateType = ""
 
 	// GoTemplate represents a go-template type.
-	GoTemplate TemplateType = "GoTemplate"
+	GoTemplate TemplateType = "go-template"
 )
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// ObjectTemplateBuilder contains configuration for creating ObjectTemplates.
+type ObjectTemplateBuilder struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// TemplateFile is references a template file
+	TemplateFile File `json:"templateFile,omitempty"`
```

Comment:
"template" in "templateFile" and "templateType" are fairly redundant. In context, it seems pretty obvious what type and file would refer to.

Reply:
Good idea. I'll come back and clean this up